src/Parser/Priest/Discipline/Modules/Spells/TwistOfFate.js

```diff
@@ -60,8 +63,32 @@ class TwistOfFate extends Analyzer {
     this.healing += calculateEffectiveHealing(event, TWIST_OF_FATE_HEALING_INCREASE);
   }
 
+  suggestions(when) {
+    if (this.hasSoulOfTheHighPriest === false) {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(SuggestionThresholds.TWIST_OF_FATE_TALENT_PERCENT.minor)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>Consider picking a different talent than <SpellLink id={SPELLS.TWIST_OF_FATE_TALENT.id}/>. Castigation will give a consistent 3-5% increase and Schism provides a significant DPS increase if more healing is not needed.</span>)
```

Comment:
Could you try to make it clear in the suggestion *why* this is recommended?

Reply:
I just realized this and it's nowhere else so there's no way you could have known, but for future suggestion instead of wrapping it in `<span>`  tags I recommend using the `Wrapper` component in `common/Wrapper`. This reduces the amount of needless elements which makes the source cleaner and more speedy.

Reply:
This does seem like more of a l2p thing for guides than something that belong to a gameplay improving suggestion, how will you ensure it's only shown when it is accurate?